Producers

```diff
@@ -383,19 +383,47 @@ type Global struct {
 	LastNameClose                  string  `json:"last_name_close"`
```

Comment:
The file name changed from `responses.go` to `Producers`, this is why automated tests fails. Please revert the file name change.

Reply:
thx